### PR TITLE
changes in css of landing page , and track page

### PR DESCRIPTION
**Pull request by sanil011**

### CODE REVIEW

**Comment by sanil011** [comment]
Screenshot 2022-11-15 at 2 47 15 AM
Screenshot 2022-11-15 at 2 47 41 AM

**Review by ank1traj — COMMENTED** [review]
We should not have a `.lock.json`, please delete it

**Comment by sanil011** [comment]
> We should not have a `.lock.json`, please delete it

deleted..

**Comment by ank1traj** [comment]
> > We should not have a `.lock.json`, please delete it
> 
> deleted..

`yarn.lock` file is still there @sanil011

**Review by ank1traj — APPROVED** [review]
LGTM

**Comment by sanil011** [comment]
> LGTM

okay , who will merge this??

**Comment by ank1traj** [comment]
> > LGTM
> 
> okay , who will merge this??

You must have rights to do so.

File: pages/tracks.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                </li>
+                <li className="text-2xl tracking-wide border-2 border-[#005550] py-6 my-6 text-white bg-[#005550] rounded-full text-center">
+                  <h2>Blockchain / Web3</h2>
+                </li>
+                <li className="text-2xl tracking-wide border-2 border-[#005550] py-6 my-6 text-white bg-[#005550] rounded-full text-center">
+                  <h2>Artificial Intelligence/Machine Learning</h2>
+                </li>
+                <li className="text-2xl tracking-wide border-2 border-[#005550] py-6 my-6 text-white bg-[#005550] rounded-full text-center">
+                  <h2>Android Development</h2>
+                </li>
```
Comment (Suvraneel) [inline pages/tracks.tsx]: Instead of redundant entries...
Please use the `.map` functionality to maintain D.R.Y format

**Review by Suvraneel — CHANGES_REQUESTED** [review]

**Review by Suvraneel — APPROVED** [review]
Since we're getting this done on priority, Imma make whatever changes needed before merging.
Also, I'll merge the PRs irrespective of approvals/requested changes as long as it maintains basic software architecture & follows principles of SOLID, DRY etc.

LGTM @sanil011  :octocat:

**Pull request merged**